0001_cars_test.cy.js

```diff
@@ -778,6 +778,125 @@ describe("Test Child:", () => {
    });
 });
 
+describe("Test dataFilter Widget:", () => {
+   it("Test Add Filter", () => {
+      //act
+      Common.RunSQL(cy, folderName, ["init_db_for_data_filter_widget.sql"]);
+      openCars();
+      cy.get(
+         "[data-cy='ABViewDataFilter_a4537949-5dd8-4596-9b35-3c854751242d filter button']"
+      )
+         .should("exist")
+         .click();
+      cy.get(
+         'div[class*="webix_view webix_control webix_el_button webix_primary"]'
+      )
+         .find(".webix_button")
+         .should("exist")
+         .contains("Add filter")
+         .click();
+      cy.get(".webix_inp_static").eq(0).should("be.visible").click();
+      cy.get('div[webix_l_id="First Name"]').contains("First Name").click();
+      cy.get(".webix_el_box")
+         .find("[type='text']")
+         .eq(5)
```

Comment:
We will need to keep an eye on this when we upgrade webix eventually. If the test fails it may just be due to small DOM changes.

Reply:
Ok James, Thank you